src/Microsoft.TestPlatform.TestHostProvider/Hosting/DefaultTestHostManager.cs

```diff
@@ -197,6 +177,67 @@ public async Task<bool> LaunchTestHostAsync(TestProcessStartInfo testHostStartIn
         };
     }
 
+    private string GetTestHostName(Architecture architecture, Framework targetFramework, PlatformArchitecture processArchitecture)
+    {
+        string testHostProcessName = "testhost";
+
+        if (targetFramework.Name.StartsWith(".NETFramework,Version=v"))
+        {
+            var targetFrameworkMoniker = "net" + targetFramework.Name.Replace(".NETFramework,Version=v", string.Empty).Replace(".", string.Empty);
```

Comment:
Logic is relatively simple but it might help to have a small comment like

```suggestion
			// Transform target framework name into moniker.
			// e.g. ".NETFramework,Version=v4.7.2" -> "net472".
            var targetFrameworkMoniker = "net" + targetFramework.Name.Replace(".NETFramework,Version=v", string.Empty).Replace(".", string.Empty);
```

Reply:
Added more comments with reasons why stuff is done this way.